db/src/Pos/DB/Rocks/Functions.hs

```diff
@@ -41,6 +41,7 @@ import qualified Database.RocksDB as Rocks
 import           System.Directory (createDirectoryIfMissing, doesDirectoryExist,
                      removeDirectoryRecursive)
 import           System.FilePath (takeDirectory, (</>))
+import qualified System.Info (os)
```

Comment:
This is already on `develop`!
https://github.com/input-output-hk/cardano-sl/blob/develop/db/src/Pos/DB/Rocks/Functions.hs#L44